frog/imports/ui/Dashboard/DashboardNav.jsx

```diff
@@ -0,0 +1,123 @@
+// @flow
+
+import * as React from 'react';
+import { withTracker } from 'meteor/react-meteor-data';
+import { withState } from 'recompose';
+
+import { withStyles } from 'material-ui/styles';
+import Drawer from 'material-ui/Drawer';
+import List from 'material-ui/List';
+import Button from 'material-ui/Button';
+
+import { Activities } from '../../api/activities';
+import { activityTypesObj } from '../../activityTypes';
+import { DashboardReactiveWrapper } from './index';
+
+const drawerWidth = 180;
+
+const styles = theme => ({
+  root: {
+    flexGrow: 1,
+    width: '100%'
+  },
+  button: {
+    floar: 'left'
+  },
+  appFrame: {
+    height: 600,
+    overflow: 'hidden',
+    zIndex: 1,
+    position: 'relative',
+    display: 'flex',
+    width: '100%'
+  },
+  toolbar: theme.mixins.toolbar,
+  drawerPaper: {
+    position: 'relative',
+    width: drawerWidth,
+    backgroundColor: theme.palette.background.default
+  },
+  content: {
+    flexGrow: 1,
+    overflow: 'auto',
+    backgroundColor: theme.palette.background.default,
+    width: `calc(100% - ${drawerWidth}px)`
+  }
+});
+
+const ActivityChoiceButton = withStyles(styles)(
```

Comment:
Instead of having button, MUI has ListItem component for List items: https://material-ui-next.com/demos/lists/#simple-list